sysbuild/Kconfig.suit

```diff
@@ -31,14 +31,75 @@ config SUIT_ENVELOPE_ROOT_TEMPLATE_FILENAME
 config SUIT_ENVELOPE_SIGN_SCRIPT
 	string "Location of SUIT sign script"
 	depends on SUIT_ENVELOPE_SIGN
+	default "modules/lib/suit-generator/ncs/sign_script.py"
 	help
 	  Python script called to sign SUIT envelope.
 	  You can use either absolute or relative path.
 	  In case relative path is used, the build system uses NRF parent directory.
 	  Script need to accept two arguments:
 	  - --input-file <STRING> - location of unsigned envelope in the build system
 	  - --output-file <STRING> - location of signed envelope to create by script
-	default "modules/lib/suit-generator/ncs/sign_script.py"
+
+choice SUIT_ENVELOPE_ENCRYPT_SCRIPT
+	prompt "Select the used SUIT encryption script"
+	default SUIT_ENVELOPE_ENCRYPT_SCRIPT_DEFAULT
+	help
+	   The SUIT encryption script is used to generate encryption artifacts for images inside a SUIT envelope.
+	   It is the "external" script that is called by the build system - it is responsible for creating
+	   various needed SUIT structures, but passes the actual encryption of the payload to a separate KMS script.
+
+config SUIT_ENVELOPE_ENCRYPT_SCRIPT_DEFAULT
+	bool "Use the default SUIT encryption script"
+	help
+	  Use the default encryption script to generate encryption artifacts for images inside a SUIT envelope.
+
+endchoice
+
+config SUIT_ENVELOPE_ENCRYPT_SCRIPT_PATH
+	string "Location of SUIT encrypt script"
+	default "${ZEPHYR_SUIT_GENERATOR_MODULE_DIR}/ncs/encrypt_script.py" if SUIT_ENVELOPE_ENCRYPT_SCRIPT_DEFAULT
+	help
+	  Python script called to generate encryption artifacts for images inside a SUIT envelope.
+	  See the help message for the default encryption script to see what arguments the script
+	  must accept.
+
+choice SUIT_ENVELOPE_KMS_SCRIPT
+	prompt "Select the used SUIT KMS script to be used by the encryption script"
+	help
+	   This is the "internal" python script that is called by the encryption script to perform the cryptographic operations,
```

Comment:
1x tab followed by 2x spaces for help text indent

Reply:
Ok, fixed